packages/code-editor/src/MonacoEditor/MonacoEditorContainer.js

```diff
@@ -104,8 +111,45 @@ export default class MonacoEditorContainer extends PureComponent {
       )
     }
 
+    if (modelSession.breadcrumb) {
+      breadcrumb = (
+        <div className='px-2 border-bottom-black text-muted topbar-breadcrumb'>
+          { modelSession.breadcrumb.map((item, index) => 
+            <>
+            <UncontrolledButtonDropdown direction='bottom'>
+            <DropdownToggle
+              size='sm'
+              color='default'
+              className='rounded-0 text-muted px-2 text-nowrap text-overflow-dots'
+              style={{ maxWidth: 240 }}
+              id='docker-version'
+            >
+              {index > 0 && <div className='breadcrumb-delimiter'></div>}
+              {item.current.name}
+            </DropdownToggle>
+            <DropdownMenu right className={this.props.size === 'sm' && 'dropdown-menu-sm'}>
+              {item.brothers.map(brother => 
+              <DropdownItem onClick={
```

Comment:
it's better to abstract the function
```suggestion
              <DropdownItem onClick={handleSetPosition}
```